src/renderers/imgui_opengl3.rs

```diff
@@ -1,3 +1,29 @@
-// TODO: Consider making local versions of gl_loader and imgui opengl renderer,
-// but this works for the time being
-pub use {gl_loader, imgui_opengl_renderer};
+use std::ffi::{c_void, CString};
+
+use once_cell::sync::OnceCell;
+use windows::core::PCSTR;
+use windows::Win32::Foundation::{FARPROC, HINSTANCE};
+use windows::Win32::Graphics::OpenGL::wglGetProcAddress;
+use windows::Win32::System::LibraryLoader::{GetProcAddress, LoadLibraryA};
+
+static mut OPENGL3_LIB: OnceCell<HINSTANCE> = OnceCell::new();
```

Comment:
Last thing and then we're good to go, `OnceCell::get_or_init` should not require mutability so I think this should be preferable:

```suggestion
static OPENGL3_LIB: OnceCell<HINSTANCE> = OnceCell::new();
```